.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+| Question      | Answer
+| ------------- | ---
+| Tickets       | <!-- URLs to GitHub or JIRA issue(s) (or N/A) -->
+| Bug fix?      | yes/no
+| New feature?  | yes/no
```

Comment:
Why should we use this instead of github labels?

Reply:
Because we're not the only ones to create PRs. There's also community.

Reply:
> Why should we use this instead of github labels?

> Because we're not the only ones to create PRs. There's also community.

and not all people have permissions to set labels (even within organization).